test/hotspot/jtreg/vmTestbase/nsk/jvmti/GetAllThreads/allthr001/allthr001.cpp

```diff
@@ -196,20 +225,33 @@ JNIEXPORT void checkInfo(JNIEnv *env, int ind) {
         return;
     }
 
+    // check unexpected threads
     for (i = 0; i < threadsCount; i++) {
-        if (!isThreadExpected(jvmti, threads[i])) {
-            num_unexpected++;
+        err = jvmti->GetThreadInfo(threads[i], &inf);
+        if (err != JVMTI_ERROR_NONE) {
+            printf("Failed to get thread info: %s (%d)\n",
+                   TranslateError(err), err);
+            result = STATUS_FAILED;
+            return;
+        }
+        if (printdump == JNI_TRUE) {
+            printf(" >>> %s", inf.name);
```

Comment:
Don't you want a newline here?

Reply:
No. This is pre-existing code.
The log looks like:
 >>> Check: 0
 >>> main >>> Reference Handler >>> Finalizer >>> Signal Dispatcher >>> Attach Listener >>> Notification Thread >>> Common-Cleaner >>> MainThread